src/components/FooterLogo.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import get from 'lodash/get'
+import { withClient } from 'cozy-client'
+
+export class FooterLogo extends React.Component {
+  state = {
+    logos: []
+  }
+
+  async componentDidMount() {
+    const context = await this.props.client
+      .getStackClient()
+      .fetchJSON('GET', '/settings/context')
+    const logos = get(context, 'data.attributes.home_logos', [])
+    this.setState({ logos })
+  }
+
+  render() {
+    const { logos } = this.state
+    const { client } = this.props
+    const rootURL = client.getStackClient().uri
+    return logos.length > 0 ? (
+      <div className="u-maw-7 u-m-auto u-pv-1 u-mt-1 u-ta-center">
+        {logos.map(logoSrc => {
+          const matches = logoSrc.match(/([\w_-]+)\..+$/)
+          const fileName = get(matches, '1', '')
+          const humanReadableName = fileName.split(/[_-]/).join(' ')
```

Comment:
Should we create a getHumanReadleNameFromSrc ? It'll be easier to understand I think

Reply:
What kind of logos are there ? It seems pretty brittle to base ourselves on the filename, cannot we add the readable name in the data itself ?